aggregation/src/idl/aggregations.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto2";
+option java_package = "fk.prof.aggregation.proto";
+option java_outer_classname = "AggregatedProfileModel";
+
+message ProfileSourceInfo {
+	optional string ip = 1;
+	optional string hostname = 2;
+	optional string processName = 3;
+	optional string zone = 4;
+	optional string instanceType = 5;
+}
+
+enum ProfileStatus {
```

Comment:
Out of these status only 3 of them, namely COMPLETED, RETRIED and ABORTED are the final status. So i am only putting these ones.